webrtc-stats.html

```diff
@@ -1971,12 +1972,21 @@
               <dd>
                 <p>
                   It is the total time each audio sample or video frame takes from the time it is
-                  received to the time it is rendered. The delay is measured from the time the
-                  first packet belonging to an audio/video frame enters the jitter buffer to the
-                  time the complete frame is sent for rendering after decoding. The average jitter
-                  buffer delay can be calculated by dividing the <code>jitterBufferDelay</code>
-                  with the <code>framesDecoded</code> (for video) or
-                  <code>totalSamplesReceived</code> (for audio).
+                  received and enters the jitter buffer to the time it exits the jitter buffer. This
```

Comment:
Just delete "and enters the jitter buffer".

Reply:
Done